packages/colibri/src/config/web_config.html

```diff
@@ -594,6 +594,20 @@ <h6 class="card-subtitle mb-2 text-muted"></h6>
                       <option value='none'>None</option>
               </select>
             </div>
+
+
+
+            <div class="mb-3">
+              <label for="documentation-general-tasks" class="form-label">
+                Include tasks:
+                <span class="markConfig badge bg-secondary" id="mark_documentation-general-tasks"></span>
+              </label>
```

Comment:
Have you used the Python script or manually?

Reply:
This part was done by hand. was there a python script that can be used for these updates?